### ISSUE TITLE
Indicators prioritisation for Accessibility

### ISSUE THREAD

**Issue by RDA-FAIR** [RDA-FAIR/FAIR-data-maturity-model-WG#31]
Dear members of the Fair data maturity model Working Group, 

As a next step towards a common set of core assessment criteria, we started to explore the prioritisation of the indicators derived from the contributions to this very repository. As a result, all the indicators have been ranked according to three degrees of priority;

- `Mandatory`: indicator MUST be satisfied for FAIRness
- `Recommended`: indicator SHOULD be satisfied, if at all possible, to increase FAIRness
- `Optional`: Indicator MAY be satisfied, but not necessarily so

In the following [Google spreadsheet](https://docs.google.com/spreadsheets/d/1mkjElFrTBPBH0QViODexNur0xNGhJqau0zkL4w8RRAw/edit#gid=0), you can see the i) indicators in their final version and ii) their priorities. 

![A](https://user-images.githubusercontent.com/47179213/62785303-ac607300-bac0-11e9-8c70-2c4d946c6f80.png)

Please have a look at the spreadsheet and let us know what are your thoughts, suggestions and concerns with regards to the ranking we attempted to make. 

Thanks in advance for your contribution!

**Comment by keithjeffery** [comment]
@Makx -
I believe we should get all the indicators defined to the satisfaction of the WG before trying to prioritise; the prioritisation may well depend on the final form of each indicator.

**Comment by makxdekkers** [comment]
@keithjeffery Yes, that's the idea. All formulations of indicators are open to review and comment by the WG. If you have any proposals, please make them in the relevant GitHub issues.

**Comment by rwwh** [comment]
Please also refer to the discussion under Findability that all DOIs (and many HANDLEs?) are actually resolving to metadata. A1-03D may fail too often to be mandatory.

**Comment by nsjuty** [comment]
A1-03D 'Data identifier resolves to a data file'. 
Are we typing identifiers at this point? I presume this is to differentiate metadata identifiers resolving to data files, but the wording will cause issues, no? If you remove 'data' from this indicator, it then becomes a simple 'resolvability' indicator. Or is it really about being able to tell if a given identifier resolves to the type of data/metadata or database record that you anticipate?

**Comment by makxdekkers** [comment]
@rwwh 


> Please also refer to the discussion under Findability that all DOIs (and many HANDLEs?) are actually resolving to metadata. A1-03D may fail too often to be mandatory.

The indicator A1-03D could be made recommended.

**Comment by makxdekkers** [comment]
@nsjuty 


> A1-03D 'Data identifier resolves to a data file'.
> Are we typing identifiers at this point? I presume this is to differentiate metadata identifiers resolving to data files, but the wording will cause issues, no? If you remove 'data' from this indicator, it then becomes a simple 'resolvability' indicator. Or is it really about being able to tell if a given identifier resolves to the type of data/metadata or database record that you anticipate?

The  principle "_A1: (meta)data are retrievable by their identifier using a standardised communications protocol_" implies that the identifier can be used to retrieve the thing (in this case the digital object) it identifies. The indicator tries to enable evaluation of this requirement. The formulation of the indicator might add precision that is not in the principle. Can you think of a better way to formulate the indicator?

**Comment by hervelh** [comment]
As for FINDABLE, there may be ‘object model’ assumptions here e.g. 
“ A1 A1-03D Data identifier resolves to a data file Mandatory”

If we assume that DOI refers to a (complex) digital object consisting of metadata and data then this not (always) practical. Could this require resolution to a digital object? 

*
How is? 

“A1 A1-01M Metadata includes information about access *conditions* Optional”

Aligned with:
“
“A1.2 A1.2-01M Metadata includes information relevant for access *control* Mandatory”

As noted under F, could manual (human-mediated?) vs automatic (machine-actionable?) be a second concentric circle of FAIR indicators/evaluation? 

*
“Available”, “accessed” and “accessible” are all used. Standardise on “accessible”?

*
Could indicators be reworded to avoid “download”? This access method seems unnecessarily specific

**Comment by makxdekkers** [comment]
@hervelh 

> If we assume that DOI refers to a (complex) digital object consisting of metadata and data then this not (always) practical. Could this require resolution to a digital object?

See also the discussion at https://github.com/RDA-FAIR/FAIR-data-maturity-model-WG/issues/30. Changing the wording from "data file" to "digital object" would allow also the (indirect) resolution of the identifier to the data resource, if it is agreed that the DOI identifies the complex object that includes or links to the metadata and data.

**Comment by makxdekkers** [comment]
@hervelh 

> As noted under F, could manual (human-mediated?) vs automatic (machine-actionable?) be a second concentric circle of FAIR indicators/evaluation?

There  is a suggestion at ://github.com/RDA-FAIR/FAIR-data-maturity-model-WG/issues/20#issuecomment-519984630 to include an indicator for human actions as a protocol.

**Comment by makxdekkers** [comment]
@hervelh 

> “Available”, “accessed” and “accessible” are all used. Standardise on “accessible”?

We'll make this consistent.

**Comment by makxdekkers** [comment]
@hervelh 

> Could indicators be reworded to avoid “download”? This access method seems unnecessarily specific

We can use 'access' instead of 'download'.

**Comment by rwwh** [comment]
"Access" suggests read-only, which may be appropriate here.

In another context I have suggested "operate on".

**Comment by keithjeffery** [comment]
In other fora the word 'action' or 'actionable' has been used.  We should surely be deprecating use of download because of network latency especially with ever-increasing sizes of digital objects.

**Comment by makxdekkers** [comment]
How about "data can be accessed manually (i.e. with human intervention)" and "data can be accessed automatically (i.e. by a computer program)"?

**Comment by makxdekkers** [comment]
@rwwh @keithjeffery Terms like "operate on", "actionable" imply that you can do something with the data resource, while the principle is about getting access to the data resource. Acting on it seems to be more in the area of reuse.
As these indicators are in the FAIR area of "Accessibility" and the principle A1 is about retrieval, not processing, I would suggest we keep "access".

**Comment by keithjeffery** [comment]
@makxdekkers I understand your reasoning.  However, various dictionary definitions of access include concepts of permission to enter/use  which sort of 'spils over' into interopration or reuse.

**Comment by hervelh** [comment]
> How about "data can be accessed manually (i.e. with human intervention)" and "data can be accessed automatically (i.e. by a computer program)"?

@makxdekkers this is always a tricky one, because if at any point in the protocol-driven process the machine (computer program etc) needs to be ‘approved’ as a fit and proper agent by a human, then the process is not entirely automated, even if every subsequent transaction can be entirely machine-machine mediated. I need to think more on this alongside the issue of machine-understandable vs actionable.

**Comment by hervelh** [comment]
@keithjeffery @rwwh @makxdekkers 
Agree with @keithjeffery that the dictionary definitions imply overspill and with @rwwh that access can imply read-only. But I suggest that within the FAIR acronym (and within the process of defining indicators)  there needs to be some philosophy of approach which seeks mutual-exclusion or at least minimises overlap. E.g. (F) The object and environment characteristics which enable and support the discovery of the existence of an object. (A) The characteristics which are important to reaching the point where access to the object is permitted (or refused) so (A) is *after* finding an object but *before* any action is taken on the object.
*
 I and R are more tricky, but perhaps something like: 
(I) The object and environment characteristics which minimise barriers to using/reusing a digital object. (R) the object and environment characteristics which support evaluation of, and define and enable the possible actions on, a digital object.

**Comment by kitchenprinzessin3880** [comment]
@makxdekkers 
may i know the difference between A1-01M (access conditions) and A1.2-01M (access control)? if you consider the [data cite schema](https://schema.datacite.org/), which properties represent this information?

**Comment by makxdekkers** [comment]
@kitchenprinzessin3880 

A1-01M is about access conditions, i.e. a metadata field that describes, for example, that the data is open access, or that it is only available for particular audiences, or that you need to login or request permission from the owner. 
A1.2-01M is about the mechanism that is available to gain access, i.e. a metadata field that links to some access control service, for example a login screen.

The information about access control under A1.2-01M is only relevant if the access conditions under A1-01M specify that access is restricted.

I cannot answer your question about the DataCite schema. I searched the file at https://schema.datacite.org/meta/kernel-4.3/doc/DataCite-MetadataKernel_v4.3.pdf, but don't see any information about access conditions or access control.

**Comment by helenp** [comment]
Flagging that usage restrictions can be encoded using an ontology and transparency of data usage restrictions is preferable to get to a login page.  Complex usage restrictions often seen in LifeSciences data mean that some usage is restricted  - e.g. for commercial entities. So a link to an access mechanism is useful, but is incomplete.

**Comment by helenp** [comment]
Data use ontology https://www.ebi.ac.uk/ols/ontologies/duo - a GA4GH standard.

**Comment by makxdekkers** [comment]
@helenp My example of pointing to a login page might indeed not be very good. However, the Data Use Ontology that you refer to seems to be more about re-use than about access. The WG discussed access conditions and re-use licences back in April (see https://www.rd-alliance.org/group/fair-data-maturity-model-wg/post/workshop-2-report) and decided that these are separate. This distinction is for example made in the Data Catalogue vocabulary ([DCAT](https://www.w3.org/TR/vocab-dcat-2/)), using `dct:accessRights` for access conditions and `dct:license` for the re-use licence.

The distinction was the reason that indicator A1-01M was added, although the FAIR principles themselves do not explicitly mention access conditions. Re-use licences fall under principle R1.1.

For access rights, there are some published vocabularies, e.g. the [Access right authority table](https://op.europa.eu/en/web/eu-vocabularies/at-concept-scheme/-/resource/authority/access-right/?target=Browse) at the EU Publications Office or the [Eprints AccessRights Vocabulary Encoding Scheme ](http://www.ukoln.ac.uk/repositories/digirep/index/Eprints_AccessRights_Vocabulary_Encoding_Scheme) at JISC. Both those vocabularies include three terms: open/public, restricted, closed/non-public.

**Comment by bahimc** [comment]
Dear contributors, 

As reminded during our [fifth workshop](https://www.rd-alliance.org/system/files/documents/20191023_FAIR_WG_Workshop_5_v0.02.pdf) in Helsinki, the **[current state of the indicators](https://docs.google.com/spreadsheets/d/1mkjElFrTBPBH0QViODexNur0xNGhJqau0zkL4w8RRAw/edit#gid=1558529297)** - and their priorities, as of early October 2019, is now frozen. The current indicators will be used in a testing phase where owners of evaluation approaches are invited to compare their approaches (questionnaires, tools) against the indicators. More information concerning the testing phase will be communicated during [workshop #6](https://www.rd-alliance.org/group/fair-data-maturity-model-wg/post/rda-fair-data-maturity-model-workshop-6-preparation), on the 4th of December. 

As such, the current set of indicators can be seen as an ‘alpha version’. In the first half of 2020, the indicators may be revised and improved, based on the results of the testing. 

Should you have any question or feedback, please reply to this comment.
